template/template.go

```diff
@@ -28,7 +29,8 @@ func Apply(contents []byte, releaseInfo ReleaseInfo, variables map[string]string
 	buf := bytes.Buffer{}
 	t.Execute(&buf, map[string]interface{}{
 		"Values":  variables,
-		"Release": releaseInfo,
+		"Release": stackInfo,
```

Comment:
Release was what I previously used and it matches the Helm style. I wanted to leave it in there in case someone was using it already. We can probably just document the Stack one now.